packages/yourloops/lib/auth/helpers.ts

```diff
@@ -27,3 +27,10 @@ export function checkPasswordStrength(password: string): CheckPasswordStrengthRe
   }
   return { onError, helperText, score };
 }
+
+
+export const genderLabels = (): Record<string, string> => ({
```

Comment:
Can't you simply have 
```
enum genderLabels {
  M = t("male"),
  F = t("female"),
}
```
if gender is not mandatory, then it can simply be set to undefined when not specified?